src/Task1/SelectGenre.jsx

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from "react";
+import "./selectGenre.css";
+const SelectGenre = () => {
+  const [show, setShow] = useState("");
+  const [clip, setClippath] = useState("polygon(50% 75%, 0 25%, 100% 25%)");
+  return (
+    <div style={{ display: "flex", justifyContent: "center" }}>
+      <div
+        id="list1"
+        className={"dropdown-check-list " + show}
+        tabindex="100"
+        style={{ width: "400px", backgroundColor: "#232323", color: "#FFFFFF" }}
+      >
+        <span
+          onClick={() => {
+            if (show === "visible") {
+              setShow("");
+              setClippath("polygon(50% 75%, 0 25%, 100% 25%)");
+            } else {
+              setShow("visible");
+              setClippath("polygon(50% 25%, 0 75%, 100% 75%)");
+            }
+          }}
+          className="anchor"
+          style={{
+            width: "96%",
+            backgroundColor: "#323232",
+            fontSize: "20px",
+          }}
+        >
+          Select Genre
+          <span style={{ clipPath: clip }} className="arrow">
```

Comment:
This `clip-path` css styles should be in css file. Mixing styles with other code is not good. Keep in mind separation of concerns principle. Generally, CSS is responsible for styles, not HTML or JS. To apply specific styles based on openness state, you can just create an additional modifier css class for open state `.is-open` and apply this class based on your state.

Reply:
I got it.